io-engine/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -417,6 +421,7 @@ impl<'n> Nexus<'n> {
 
         match job_state {
             RebuildState::Completed => {
+                self.event(Some(EventAction::RebuildEnd)).generate();
```

Comment:
Should we also generate this event for stopped and failed rebuilds?

Reply:
Yes, done.